1.1/geo.ttl

```diff
@@ -580,6 +589,53 @@ geosparql:
 	skos:prefLabel "spatial dimension"@en ;
 .
 
+:metricSpatialResolution
+	a owl:DatatypeProperty ;
+	rdfs:domain :Geometry ;
+	rdfs:range xsd:double ;
```

Comment:
I prefer `xsd:decimal` - it allows arbitrary precision and is 'honest' in data transfer.